process/accounts/accounts_processor_test.go

```diff
@@ -164,6 +163,27 @@ func TestAccountsProcessor_ProcessAccounts_SevenAddresses_ExpectSevenAccounts(t
 	checkProcessedAccounts(t, addresses, ret)
 }
 
+func TestAccountsProcessor_ProcessAccounts(t *testing.T) {
+	t.Parallel()
+
+	ap, _ := accounts.NewAccountsProcessor(&mock.ShardCoordinatorMock{})
+
+	alteredAccountsMap := map[string]*indexer.AlteredAccount{
+		"adr0": {
+			Balance: "",
+		},
+	}
+
+	tx1 := &schema.Transaction{
+		Receiver: []byte("adr0"),
+	}
+
+	res := ap.ProcessAccounts(alteredAccountsMap, []*schema.Transaction{tx1}, []*schema.SCResult{}, []*schema.Receipt{})
+
+	require.Len(t, res, 1)
+	require.Equal(t, big.NewInt(0).Bytes(), res[0].Balance)
```

Comment:
[optional, just to check that the processed address is the one that we expect]

```suggestion
	require.Equal(t, &schema.AccountBalanceUpdate{
		Address: []byte("adr0"),
		Balance: big.NewInt(0).Bytes(),
		Nonce:   0,
	}, res[0])
```

Reply:
I cannot use the generateAddresses/checkProcessedAccounts because they will set the balance as `"0"`, and I need `""`